@jamesgeorge007 linting errors:

```
/tmp/storybook/lib/cli/bin/generate.js
  75:61  error    'cmd' is already declared in the upper scope  no-shadow
  75:68  error    Unexpected dangling '_' in '_name'            no-underscore-dangle
  77:5   warning  Replace `↹` with `··`                         prettier/prettier
```

--- a/lib/cli/bin/generate.js
+++ b/lib/cli/bin/generate.js
@@ -70,11 +70,15 @@ if (process.argv[1].includes('getstorybook')) {
       });
     });
 
-  program.command('*', { noHelp: true }).action(cmd => {
-    logger.error(' Invalid command: %s.\n See --help for a list of available commands.', cmd);
-    const suggestion = didYouMean(cmd, program.commands.map(cmd => cmd._name));
+  program.command('*', { noHelp: true }).action(invalidCmd => {
+    logger.error(
+      ' Invalid command: %s.\n See --help for a list of available commands.',
+      invalidCmd
+    );
+    // eslint-disable-next-line
+    const suggestion = didYouMean(invalidCmd, program.commands.map(cmd => cmd._name)); 
     if (suggestion) {
-    	logger.log(`\n Did you mean ${suggestion}?`);
+      logger.log(`\n Did you mean ${suggestion}?`);
     }
     process.exit(1);
   });

